www/posts/2020-05-25-v2-release-day.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: layouts/post.njk
+---
+
+After 40+ beta versions & release candidates we are so excited to introduce **Snowpack 2.0**, featuring:
+
+- A bundle-free dev environment that starts up in 50ms or less.
+- Application build tooling that stays fast as your codebase grows.
+- [Bundled production-optimized builds](/#snowpack-build) (powered by Webpack or Parcel)
+- [Hot Module Replacement (HMR)](/#hot-module-replacement) for React, Preact, Vue, Svelte & more.
+- Out-of-the-box support for TypeScript, JSX, CSS Modules & more.
+- [Custom build scripts](/#build-scripts) & [third-party plugins](/#build-plugins) to connect your favorite tools.
+
+For most Snowpack v1.x applications, all you need to do is run the following command and Snowpack 2.0 will walk you through updating any outdated configuration:
```

Comment:
I would move "upgrade steps" to a separate header later on, this is a less important detail for new readers (and old users will read through regardless)